### DISCUSSION TITLE
row reorder [Sep 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3908] [Discord Threads]
> 🙋‍♂️ **benjifrank**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### not supported currently  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020878403206926367)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by asasn** [comment]
Request support for this feature, it's really important.

**Comment by d41ly** [comment]
absolutely agreed, this is one of the crucial user experience features

**Comment by ioannismihailidis** [comment]
Please add this feature! Also options to insert an empty row between rows. Similar to Excel/Spreadsheet.
